src/Atmos/Model/AtmosModel.jl

```diff
@@ -134,6 +135,8 @@ struct AtmosModel{FT, PS, PR, O, RS, T, TC, HD, VS, M, P, R, S, TR, DC} <:
     source::S
     "Tracer Terms (Equations for dynamics of active and passive tracers)"
     tracers::TR
+    "GCM Information"
+    gcminfo::GI
```

Comment:
`gcminfo`? Why is this a submodel of the `AtmosModel`? This is not a physics model, and is counter-intuitive if we're using a single stack / LES.

Reply:
The `gcminfo` is used to store external source terms to force the LES / single stack. Maybe a better name would be `forcing`? (But that's also confusing because radiation, source, etc. are forcings too.)

Reply:
Why can't we just add sources like we already have?

Reply:
I think it's because we would like to read the netcdf during the initialization, store the forcing terms in `aux`, and compute the source terms (e.g. subsidence tendency) at each timestep. And in the future the forcing may be a function of time. @akshaysridhar can correct me. Is there an easy way to use the current `source` structure?

Reply:
Currently we're using this when computing DG-gradients of the GCM state; maybe `gcminfo` is a misnomer and it should be `modeldata` or similar, but it's being used as a way to store / pass around the gcm-states [which eventually will be time dependent]

Reply:
@charleskawczynski does this sound good? or is there a way to use `source` to do this?

Reply:
I need to take a closer look at what’s being done. It may be that GCMInfo is just a bad name

Reply:
This Atmos sub-model isn’t adding any tendencies . It kind of looks like this is just adding GCM-specific diagnostics. @kpamnany can we add the GCM-info diagnostics without adding all of these aux/grad fields?

Reply:
These are not GCM diagnostics. These are information from an external forcing file, and the tendencies are added in the driver `experiments/AtmosLES/cfsite_hadgem2-a_07_amip.jl`. See the `EnergyTendency`, `MoistureTendency`, and `SubsidenceTendency` in the driver for example.

Reply:
Ok, then we should probably just come up with a better name for `gcminfo` if it's introducing tendencies.